src/script/redmine-upkeep.py

```diff
@@ -112,8 +111,10 @@ def process(self, msg, kwargs):
 log.addHandler(log_stream)
 log.setLevel(logging.INFO)
 
-def gitauth():
-    return (GITHUB_USER, GITHUB_TOKEN)
+GITHUB_HEADERS = {
+    "Authorization": f"Bearer {GITHUB_TOKEN}",
+    "X-GitHub-Api-Version": "2022-11-28",
+}
```

Comment:
Although there seems not to be an official Python client for the GH REST API, [this one has wide adoption](https://github.com/PyGithub/PyGithub). Just to avoid hard-coding low-level details like this.